docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -49,22 +115,143 @@ GET /_search
 --------------------------------------------------
 // CONSOLE
 
-The query `path` points to the nested object path, and the `query`
-includes the query that will run on the nested docs matching the
-direct path, and joining with the root parent docs. Note that any
-fields referenced inside the query must use the complete path (fully
-qualified).
-
-The `score_mode` allows to set how inner children matching affects
-scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
+Adding the option `score_mode` allows you to set how inner children matching affects scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
 `max` and `none`.
 
-There is also an `ignore_unmapped` option which, when set to `true` will
-ignore an unmapped `path` and will not match any documents for this query.
-This can be useful when querying multiple indexes which might have different
-mappings. When set to `false` (the default value) the query will throw an
-exception if the `path` is not mapped.
+The `ignore_unmapped` option set to `true` will ignore an unmapped `path` and will not match any documents for this query. This can be useful when querying multiple indexes which might have different mappings. Set to `false` (the default value) the query will throw an exception if the `path` is not mapped.
+
+==== Multi-level nesting
 
-Multi level nesting is automatically supported, and detected, resulting
-in an inner nested query to automatically match the relevant nesting
+Multi-level nesting is automatically supported and detected, resulting
+in an inner nested query that automatically matches the relevant nesting
 level (and not root) if it exists within another nested query.
+
+To set up a multi-level nested query, use this example. Notice that each level within the nest is separated by a dot (name.state.city).
+
+[source,js]
+--------------------------------------------------
+PUT my_index
```

Comment:
I would name this index `customers`, to indicate what the index hold. The mapping can't have a name any more, so the index name should indicate what this index holds.